### PR TITLE
Add link to online documentation to the README

### PR DESCRIPTION
**Pull request by mhanberg**
Also lightly cleans up the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -146,22 +144,23 @@ Please see the [examples](https://github.com/randym/axlsx/tree/master/examples/e
...
 There is much, much more you can do with this gem. If you get stuck, grab me on IRC or submit an issue to GitHub. Chances are that it has already been implemented. If it hasn't - let's take a look at adding it in.
 
-#Documentation
---------------
+## Documentation
 
 This gem is 100% documented with YARD, an exceptional documentation library. To see documentation for this, and all the gems installed on your system use:
 
-     gem install yard kramdown
+```
```
Comment (developerfab) [inline README.md]:
```md
```bash
...
```

WDYT?

Comment (mhanberg) [reply README.md]: done

**Comment by mhanberg** [comment]
bump @randym

**Comment by noniq** [comment]
Thanks @mhanberg, the rdoc link has already been included in the README of caxlsx (https://github.com/caxlsx/caxlsx/commit/2cdaed01).

**Pull request closed without merging**